services/servers/web/routers/admin/methods/channelMemesStatePut.js

```diff
@@ -0,0 +1,16 @@
+import { setChannelMemesState } from '../../../../utils/index.js';
+import { setLogAction } from '../utils/index.js';
+
+export const channelMemesStatePut = async (req, res) => {
```

Comment:
Where's the authentication?

Reply:
It's on top of the entire adminRouter (/admin/* methods).